assets/css/style.css

```diff
@@ -312,9 +312,10 @@ body {
 .projectimages {
```

Comment:
- [ ] You have done a good job with your popup window :muscle: but there is an issue with the style because the (X) icon is showing almost in the middle of the content. I would suggest fixing this style issue in order to match the Figma design :wink: :raised_hands:

| Mobile Popup | Desktop Popup | Figma design |
| --- | --- | --- |
| ![image](https://user-images.githubusercontent.com/81522435/146050875-9009ec40-45d2-4e91-b903-94fc3fba6788.png) | ![image](https://user-images.githubusercontent.com/81522435/146051059-5d1ffa6b-d335-4b2e-8b5c-a0faeddd8255.png) | ![image](https://user-images.githubusercontent.com/81522435/146051109-16c8430d-c626-4c4e-b79a-4228124b2c8d.png) |

Reply:
thanks, I am working on it.